src/helpers/misc.js

```diff
@@ -116,6 +116,36 @@ const dirCheckSync = (dir) => {
 
 };
 
+/**
+ * Function returns an array that contains unique topics types. Values are taken from argv.topics
+ * -t 'services:serviceOne,tools:toolOne,services:serviceTwo' will return: [ 'services', 'tools' ]
+ * @param  {String} [message] - argv.topics string
+ * @return {Array} - Array with unique topic types
+ */
+function uniqTopicTypes(message) {
+  if (!message) return [];
```

Comment:
It will never happen in my scenario, cause you are not able to run gulp with flag -t without giving topics. After a discussion with Mariusz it was clear that it makes sense to add such a protection in case someone else will use this function.